examples/custom-query-parsing/src/@types/index.d.ts

```diff
@@ -0,0 +1,9 @@
+export type Nullable<T> = T | null | undefined;
```

Comment:
Why do we need this file? It looks like `@types/jsurl.d.ts` already declares everything it needs, right?